app/org/sagebionetworks/bridge/dynamodb/DynamoTask.java

```diff
@@ -56,111 +68,149 @@ public TaskStatus getStatus() {
             return TaskStatus.FINISHED;
         } else if (startedOn != null) {
             return TaskStatus.STARTED;
-        } else if (expiresOn != null && DateTime.now().isAfter(expiresOn)) {
-            return TaskStatus.EXPIRED;
-        } else if (scheduledOn != null && DateTime.now().isBefore(scheduledOn)) {
-            return TaskStatus.SCHEDULED;
+        }
+        if (timeZone != null) {
+            DateTime now = DateTime.now(timeZone);
+            if (localExpiresOn != null && now.isAfter(getExpiresOn())) {
+                return TaskStatus.EXPIRED;
+            } else if (localScheduledOn != null && now.isBefore(getScheduledOn())) {
+                return TaskStatus.SCHEDULED;
+            }
```

Comment:
With the check for a null timeZone now part of the getters, these can be used directly to assign to local variables and then test. Hopefully clearer.